UPGRADING.md

```diff
@@ -4,11 +4,19 @@ This document describes breaking changes and how to upgrade. For a complete list
 
 ## v2
 
-No major changes to the API.
+No major changes to the `level` API.
 
 The reasoning behind a new major is based on updating to `level-packager@~2.0.0` and `leveldown@~2.0.0`. Where a major rewrite of the `levelup` API was done. Also, encodings are now taken care of by `encoding-down`, which comes bundled inside `level-packager`. This was previously handled by `levelup`.
```

Comment:
> Also, encodings are now taken care of by `encoding-down`

I think this might affect third party libraries used in relation like [subleveldown](https://github.com/mafintosh/subleveldown)

Reply:
You're right. See https://github.com/Level/community/issues/14 and https://github.com/mafintosh/subleveldown/pull/7.